api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/CommonQueries.java

```diff
@@ -88,9 +88,9 @@ public static String getPatientsOnTbTreatmentQuery(
             + "WHERE pp.program_id=${5} "
             + "  AND pp.location_id=  :location  "
             + "  AND pp.date_enrolled BETWEEN   date_sub(:endDate, INTERVAL 7 MONTH) AND   :endDate "
```

Comment:
Please update the javadocs where this method is used, and add the correct period of evaluation (endDate - 7 months)
Find the Cohort definition where this method is being used **CommonCohortQueries.getPatientsOnTbTreatment()**